app/models/setting/katello.rb

```diff
@@ -10,6 +10,7 @@ def self.load_defaults
         self.set('katello_default_user_data', N_("Default user data for new Operating Systems"), 'Katello Kickstart Default User Data'),
         self.set('katello_default_PXELinux', N_("Default PXElinux template for new Operating Systems"), 'Kickstart default PXELinux'),
         self.set('katello_default_iPXE', N_("Default iPXE template for new Operating Systems"), 'Kickstart default iPXE'),
+        self.set('katello_default_atomic_provision', N_("Default provisioning template for new Atomic Operating Systems"), 'Katello Atomic Kickstart Default'),
```

Comment:
Would we want to use "Atomic Kickstart Default" instead of "Katello Atomic Kickstart Default" or would that conflict with an existing setting?  I wanted to confirm, since it seems like we did not prepend all of the settings with "Katello ".